unlock-app/src/__tests__/reducers/accountReducer.test.js

```diff
@@ -10,7 +10,7 @@ describe('account reducer', () => {
   const balance = '1337'
 
   it('should return the initial state', () => {
-    expect(reducer(undefined, {})).toEqual({ 'locks': [], transaction: null })
+    expect(reducer(undefined, {})).toEqual({ 'locks': [], transactions: {all: {}, lastUpdated: 0, latest: null} })
```

Comment:
I am asking but I have no idea if maybe it wouldn't be better to have multiple reducers, once for `transactions`, one for `lastUpdatedTransaction` and one for `latestTransaction`?

Reply:
This would avoid nesting things?

Reply:
That could well be - I'd like to use this as a starting point if that makes sense to you? Then we can work towards bringing more to the top level.